app/i18n/locales/de.js

```diff
@@ -383,20 +429,38 @@ export default {
 	Video_call: 'Videoanruf',
 	View_Original: 'Original anzeigen',
 	Voice_call: 'Sprachanruf',
+	Websocket_disabled: 'Websockets sind auf diesem Server nicht aktiviert.\n{{contact}}',
 	Welcome: 'Herzlich willkommen',
 	Welcome_to_RocketChat: 'Willkommen bei Rocket.Chat',
 	Whats_your_2fa: 'Wie ist dein 2FA-Code?',
+	Without_Servers: 'Ohne Server',
 	Yes_action_it: 'Ja, {{action}}!',
 	Yesterday: 'Gestern',
 	You_are_in_preview_mode: 'Sie befinden sich im Vorschaumodus',
 	You_are_offline: 'Sie sind offline',
 	You_can_search_using_RegExp_eg: 'Sie können mit RegExp suchen. z.B. `/ ^ text $ / i`',
-	You_colon: 'Sie:',
+	You_colon: 'Sie: ',
 	you_were_mentioned: 'Sie wurden erwähnt',
-	you: 'sie',
+	you: 'Sie',
```

Comment:
I think we prefer this with lower case.

Reply:
Why? The formal „Sie“ is always written with uppercase „S“. 

The informal „du“ can be written with „d“ (newer style) or „D“ (traditional).